Logstash/logstash-input-azurewadtable/lib/logstash/inputs/azurewadtable.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 require "logstash/inputs/base"
 require "logstash/namespace"
-
+require "time"
```

Comment:
Interesting, it's been working for a long time without this require statement. Do you know under what circumstances  that this becomes a problem?